src/WorkQueue/ColumnDefs/NPH/moduleConsentStatus.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\WorkQueue\ColumnDefs\NPH;
+
+use App\WorkQueue\ColumnDefs\defaultColumn;
+
+class moduleConsentStatus extends defaultColumn
+{
+    public function getColumnDisplay($data, $dataRow): string
```

Comment:
Could you add return types for these parameters and in other places as well?

Reply:
I don't know of a good way to type these because the interface would have to be typed as well. The interface would need to be able to handle any sort of layout of the data (array, object, string, int, etc) as well as the datarow. I think the return type is enough since the other code doesn't care what its passing into these objects and only cares that strings are returned out of it. It's up to each individual implementation of a column to sort through its data and get what is trying to be displayed.

Reply:
Gotcha!